### ISSUE TITLE
Large text sizes getting clipped on Login Screen

### ISSUE THREAD

**Issue by Ecarrion** [woocommerce/woocommerce-ios#2924]
**Describe the bug**

The "continue with wordpress.com" is getting clipped when launching the app with a big text size. 

**Screenshots**
![image](https://user-images.githubusercontent.com/562080/95088491-a6781900-06e8-11eb-8909-b3e26b4cb9fa.png)

**Expected behavior**
The whole text should be visible.

**Mobile Environment**
Please include:
- Device: IPhone 7 Plus / iOS 14

**Comment by Ecarrion** [comment]
@Garance91540 Should we force the button to spawn over two lines?

**Comment by Garance91540** [comment]
> @Garance91540 Should we force the button to spawn over two lines?

How would that look?

**Comment by Ecarrion** [comment]
@Garance91540 Apparently this is fixed in the last testflight release but it does not look very good. 

![IMG_8731](https://user-images.githubusercontent.com/562080/95213774-ce34b300-07b4-11eb-9383-999908d39e69.PNG)

Also, that button seems to be part of a different project([WordpressAuthenticator](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS)).

**Comment by Garance91540** [comment]
mmm yeah, I'd definitely not consider it fixed  😂

Is it possible to flag this in the WP Authenticator project?

**Comment by Ecarrion** [comment]
@jaclync  Since you worked on this area lately, could you confirm that this issue is from [Wordpress Authenticator](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS) and not because of any customization that we do?

**Comment by jaclync** [comment]
yes, this login sheet is in the WP authenticator library and we don't have customizations over these strings. All of our authenticator customizations are in `AuthenticationManager`:

https://github.com/woocommerce/woocommerce-ios/blob/51f5e352e8f089d85540f14622cd5adaaa20a38d/WooCommerce/Classes/Authentication/AuthenticationManager.swift#L43-L91

**Comment by jaclync** [comment]
I see that WPiOS has already moved to unified login UX so that they don't use the login sheet anymore. We could probably address this issue when we implement unified login (cc @ctarda - Aaron mentioned you'd be taking on this!)

**Comment by Ecarrion** [comment]
Closing this as this screen is not longer in the app. Unified Login replaced it.
